stdlib/sys.mli

```diff
@@ -227,11 +227,12 @@ external poll_actions : unit -> unit = "%poll"
 
 (** {1 Signal handling} *)
 
+type signal = int
```

Comment:
A doc string is missing here. This would be a good place to document OCaml's signal renumbering business. Something like:

```ocaml
(** The type for signal numbers. 

    This is either a platform independent negative number for those signals 
    that OCaml recognizes or a positive number for a platform dependent 
    signal number. The function {!signal_of_int} converts known platform dependent 
    number to independent ones.
```